packages/cli/src/util/get-latest-version/get-latest-worker.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * This file is spawned in the background and checks npm for the latest version
+ * of the CLI, then writes the version to the cache file.
+ *
+ * NOTE: Since this file runs asynchronously in the background, it's possible
+ * for multiple instances of this file to be running at the same time leading
+ * to a race condition where the most recent instance will overwrite the
+ * previous cache file resetting the `notified` flag and cause the update
+ * notification to appear for multiple consequetive commands. Not the end of
+ * the world, but something to be aware of.
+ */
+
+const https = require('https');
+const { mkdirSync, writeFileSync } = require('fs');
+const {
+  access,
+  mkdir,
+  readFile,
+  unlink,
+  writeFile,
+} = require('node:fs/promises');
```

Comment:
Any reason you're using `node:` prefix here but not the others?

Reply:
Copy and pasted from the Node.js docs. Didn't really give it much thought. I removed the `node:`, but I think we should eventually add it everywhere.